examples/paraview/circle_test_paraview_catalyst.py

```diff
@@ -0,0 +1,143 @@
+# script-version: 2.0
+# Catalyst state generated using paraview version 5.10.1
+
+#### import the simple module from the paraview
+from paraview.simple import *
+#### disable automatic camera reset on 'Show'
+paraview.simple._DisableFirstRenderCameraReset()
+
+# ----------------------------------------------------------------
+# setup views used in the visualization
+# ----------------------------------------------------------------
+
+# get the material library
+materialLibrary1 = GetMaterialLibrary()
+
+# Create a new 'Render View'
+renderView1 = CreateView('RenderView')
+renderView1.ViewSize = [1132, 679]
+renderView1.InteractionMode = '2D'
+renderView1.AxesGrid = 'GridAxes3DActor'
+renderView1.CenterOfRotation = [5.0, 5.0, 0.0]
+renderView1.StereoType = 'Crystal Eyes'
+renderView1.CameraPosition = [5.0, 5.0, 27.320508075688775]
+renderView1.CameraFocalPoint = [5.0, 5.0, 0.0]
+renderView1.CameraFocalDisk = 1.0
+renderView1.CameraParallelScale = 7.0710678118654755
+renderView1.BackEnd = 'OSPRay raycaster'
+renderView1.OSPRayMaterialLibrary = materialLibrary1
+
+SetActiveView(None)
+
+# ----------------------------------------------------------------
+# setup view layouts
+# ----------------------------------------------------------------
+
+# create new layout object 'Layout #1'
+layout1 = CreateLayout(name='Layout #1')
+layout1.AssignView(0, renderView1)
+layout1.SetSize(1132, 679)
+
+# ----------------------------------------------------------------
+# restore active view
+SetActiveView(renderView1)
+# ----------------------------------------------------------------
+
+# ----------------------------------------------------------------
+# setup the data processing pipelines
+# ----------------------------------------------------------------
+
+# create a new 'PVD Reader'
+circle_grid_pvbatchpvd = PVDReader(registrationName='circle_grid_pvbatch.pvd', FileName='/data/tjotaha/src/sparta_work/test_suite/sparta/build/examples/circle/circle_grid_pvbatch.pvd')
```

Comment:
@tjotaha this path is hard-coded, does this file need to be added to the /examples dir?

Reply:
From @tjotaha: ParaView should override that hard-coded path with the data coming from Catalyst when the script executes.